### PR TITLE
docs: improve consistency of array typings; introduce typedefs category

### PR DESCRIPTION
**Pull request by Lange**
**Might not be ready to merge yet, please review carefully!**

This is part of the ongoing work in https://github.com/haganbmj/obs-websocket-js/pull/126

### CODE REVIEW

**Comment by Palakis** [comment]
Thanks guys, gonna have a look at this in the coming days.

File: docs/generated/comments.json [hunk 1]
```diff
@@ -2960,7 +2952,7 @@
         "subheads": [],
         "description": "List all sources available in the running OBS instance",
         "return": [
-          "{Array of Objects} `sources` Array of sources as objects",
+          "{Array<Object>} `sources` Array of sources as objects",
```
Comment (Palakis) [inline docs/generated/comments.json]:
```suggestion
          "{Array<Object>} `sources` Array of sources",
```
Can also be applied to other instances of this.

**Comment by Palakis** [comment]
@Lange 
> Might not be ready to merge yet, please review carefully!

Do this qualify as a WIP?

**Comment by Lange** [comment]
@Palakis I think this is ready to be merged (via GitHub's "squash & merge" option).

**Comment by Palakis** [comment]
@Lange @haganbmj thanks guys

**Pull request merged**